commons-statistics-distribution/src/test/java/org/apache/commons/statistics/distribution/ContinuousDistributionAbstractTest.java

```diff
@@ -494,12 +494,7 @@ void testDensityIntegrals() {
         final double tol = 1e-9;
         final BaseAbstractUnivariateIntegrator integrator =
             new IterativeLegendreGaussIntegrator(5, 1e-12, 1e-10);
-        final UnivariateFunction d = new UnivariateFunction() {
-                @Override
-                public double value(double x) {
-                    return distribution.density(x);
-                }
-            };
+        final UnivariateFunction d = x -> distribution.density(x);
```

Comment:
Can this be a function reference: `distribution::density`

Reply:
Yes. changed